go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/hyperjumptech/grule
+module github.com/hyperjumptech/grule-engine
```

Comment:
Ah, I entirely forgot this due to the go module reset. Could you change this to `github.com/hyperjumptech/grule-rule-engine` ?